docs/docs/rest-api/public/api/v2/types/queue.raml

```diff
@@ -0,0 +1,35 @@
+#%RAML 1.0 Library
+uses:
+  apps: app.raml
+  pods: pod.raml
+types:
+  QueueDelay:
+    properties:
+      timeLeftSeconds?:
+        type: number
+        format: int32
+      overdue?:
+        type: boolean
+  QueueObject:
+    properties:
```

Comment:
need a discriminator and discriminator value if you're going to explicitly list the base type (not required).